monailabel/scribbles/transforms.py

```diff
@@ -40,7 +40,15 @@ def _fetch_data(self, data, key):
         if key not in data.keys():
             raise ValueError(f"Key {key} not found, present keys {data.keys()}")
 
-        return data[key].array if isinstance(data[key], MetaTensor) else data[key]
+        return data[key].array.copy() if isinstance(data[key], MetaTensor) else data[key].copy()
```

Comment:
r u sure u want to copy?

Reply:
What do you recommend as the best alternative here?

This is to address logical error described here: #844 
If I dont copy then the arrays are passed as a reference. Modifying things in calling function could lead to changes in data[key] location too which could possible lead to possible logical errors...